src/store/message.ts

```diff
@@ -16,6 +16,9 @@ interface MessageState {
   previewMessages: Map<string, Message>;
   setPreviewMessage: (key: string, message: Message) => void;
   setPreviewMessages: (previewMessages: Map<string, Message>) => void;
+  reset: () => void;
+  selectedProfile: string;
```

Comment:
Suggestion: Can we name this `selectedProfileId` so it's not confused as a full profile? Or save the full profile instead of the id -- either one!